lte/gateway/c/session_manager/LocalSessionManagerHandler.cpp

```diff
@@ -46,6 +47,7 @@ LocalSessionManagerHandlerImpl::LocalSessionManagerHandlerImpl(
 void LocalSessionManagerHandlerImpl::ReportRuleStats(
     ServerContext* context, const RuleRecordTable* request,
     std::function<void(Status, Void)> response_callback) {
+  set_sentry_transaction("ReportRuleStats");
```

Comment:
so besides crashes you want to report transactions?

Isn't it the same as the metrics/events we already track?

Reply:
The transaction isn't an event on its own. It's more like a tag. So if we get a crash while handling ReportRuleStats, it'll just say it was during the ReportRuleStats transaction.